README.md

```diff
@@ -45,7 +39,6 @@ The individual libraries are developed to be agnostic about the utilized [Distri
 - [API Reference](https://docs.rs/identity_iota/latest/identity_iota/): Package documentation (cargo docs).
```

Comment:
Would be nice to conveniently link to Rust API docs and Wasm API docs.
Same with examples maybe?

Reply:
How do you mean?

Reply:
For example:

```
- [Rust API Reference](docs.rs...)
- [Wasm API Reference](wiki.iota.org...)
```

Reply:
gotcha